client/src/components/overview/overview-components/Stars.jsx

```diff
@@ -19,7 +27,9 @@ export default function OverviewStars({ stars, reviewCount }) {
           starHoverColor="yellow"
           name='rating'
         />
-        <div style={{cursor: 'pointer', textDecoration: 'none', color: '#6699cc'}}>Read all {reviewCount} reviews</div>
+        <div onClick={() => {
+          handleScroll("reviews")
+        }}style={{cursor: 'pointer', textDecoration: 'none', color: '#6699cc'}}>Read all {reviewCount} reviews</div>
       </StyledOverviewStars>
   );
 }
```

Comment:
This was where I added the ability to clink on the reviews count in the overview that would scroll down to the reviews